src/app/components/planner-board/planner-board.component.less

```diff
@@ -4,7 +4,7 @@
 
 .work-item-page {
   position: relative;
-  .calc(height; "100vh - 60px");
+  height: ~"calc(e('100vh - 60px'))";
```

Comment:
why the extra `e` in here? it should work as just `height: ~'calc(100vh - 60px)';`